backends/ref/ceed-ref-restriction.c

```diff
@@ -28,8 +28,8 @@ static inline int CeedElemRestrictionApplyStridedNoTranspose_Ref_Core(CeedElemRe
     // CPU backend strides are {1, elem_size, elem_size*num_comp}
     // This if branch is left separate to allow better inlining
     for (CeedSize e = start * block_size; e < stop * block_size; e += block_size) {
-      CeedPragmaSIMD for (CeedSize k = 0; k < num_comp; k++) {
-        CeedPragmaSIMD for (CeedSize n = 0; n < elem_size; n++) {
```

Comment:
Rationale for reordering these lines should probably be separate from removing the pragmas.

Reply:
Ahh, oops, was trying some things and forgot to separate these commits.

My thinking here is that num_comp is one of the things we specialize on, so this pushes all of the loops that can be statically unrolled together towards the inside.